src/object_based/extent.cpp

```diff
@@ -58,7 +58,8 @@ namespace ear {
   }
 
   std::pair<double, double> azimuthElevationOnBasis(
```

Comment:
It could be, but I think this function is removed by this PR? (assuming you're referring to `azimuthElevationOnBasis`).

Reply:
I just had another look over it all though, and made some more functions const.

Caches in `SpreadingPannerHTM` have to be mutable -- these are essentially temporaries which are stored to avoid allocation.